modules/hitl2/src/views/full/Components/Conversation.tsx

```diff
@@ -27,6 +23,7 @@ const Conversation: FC<Props> = props => {
   async function handleAssign() {
     try {
       const escalation = await api.assignEscalation(props.escalation.id)
+      // tslint:disable-next-line: no-floating-promises
```

Comment:
why not await ?

Reply:
Again, not my code, just muted the linting so I wouldn't see it